### ISSUE TITLE
Add option to not sync with other windows?

### ISSUE THREAD

**Issue by potato20** [funkensturm/ember-local-storage#276]
at the moment i'm just overriding `_addStorageListener()` and making it do another but was wondering if you guys were planning to add an option to skip that.

**Comment by fsmanuel** [comment]
Hey @potato20 thanks for the feedback. Do you want to turn it off globally, per storage or more dynamically like for IE users?

**Comment by potato20** [comment]
My use case isn't for IE users, but a global and per-storage solution seems feasible.

**Comment by lsg-richard** [comment]
Yes having the option to turn this off would be good. I couldn't see how without forking - is there a better way? Adding `_addStorageListener: () => null` to my storage class didn't seem to do it.

**Comment by fsmanuel** [comment]
@lsg-richard How did you do it on your fork?

**Comment by lsg-richard** [comment]
Just commenting out adding the listener seemed to do it:
https://github.com/lightspeedgraphics/ember-local-storage

**Comment by lsg-richard** [comment]
Actually no, don't do this - it breaks things. Computed properties depending on a value in a local storage object started having sync issues after this change.

Need to find the correct way to do this.

**Comment by fsmanuel** [comment]
@lsg-richard I'll also investigate how we can do it!
